steampy/client.py

```diff
@@ -254,3 +253,7 @@ def fetch_price(self, item_hash_name: str, game: GameOptions, currency: str = Cu
                   'appid': game.app_id,
                   'market_hash_name': item_hash_name}
         return self._session.get(url, params=params).json()
+
+
+class SevenDaysHoldException(Exception):
+    """Account has logged in a new device and can't trade for 7 days"""
```

Comment:
"""Account has logged in from a new device and can't trade for 7 days"""

You are sure that every time user log in he won't have this exception.
Maybe steam thinks that every bot session is from new device?